kvmagent/kvmagent/test/localstorage_testsuite/test_data_volume_1.py

```
@@ -0,0 +1,116 @@
+        _, virtio1_vol = vm_utils.attach_virtio_volume_to_vm(vm_uuid, virtio1_vol_uuid, virtio1_vol_path)
+        _, virtio_scsi1_vol = vm_utils.attach_virtio_scsi_volume_to_vm(vm_uuid, virtio_scsi1_vol_uuid,
+                                                                       virtio_scsi1_vol_path)
+        _, virtio2_vol = vm_utils.attach_virtio_volume_to_vm(vm_uuid, virtio2_vol_uuid, virtio2_vol_path)
+        _, virtio_scsi2_vol = vm_utils.attach_virtio_scsi_volume_to_vm(vm_uuid, virtio_scsi2_vol_uuid,
+                                                                       virtio_scsi2_vol_path)
+
+        rsp = vm_utils.check_volume(vm_uuid, [virtio1_vol, virtio_scsi1_vol, virtio2_vol, virtio_scsi2_vol])
+        self.assertTrue(rsp.success)
+
+        def create_snapshot_job(vol_uuid, vol_path, memory=False):
+            snapshotJobs = {
```

Ruff (0.12.2)

Local variable `snapshotJobs` is assigned to but never used

Remove assignment to unused variable `snapshotJobs`

(F841)